src/main/java/frc/robot/commands/arm/ArmFloorCMD.java

```diff
@@ -10,12 +11,19 @@ public class ArmFloorCMD extends SequentialCommandGroup {
 
   public ArmFloorCMD(Arm arm, IntakeRelease intakeRelease) {
     addCommands(
-        new ArmPositionCMD(
+        new ArmRotateCMD(
             arm,
-            () ->
-                intakeRelease.wantsCone()
-                    ? ArmPositionConstants.CONE_FLOOR
-                    : ArmPositionConstants.CUBE_FLOOR),
-        new IntakeAndRaise(arm, intakeRelease));
+            intakeRelease.wantsCone()
```

Comment:
This needs to be in a lambda or it will be evaluated once during RobotContainer construction.